Hardcore.lua

```diff
@@ -721,6 +750,27 @@ function Hardcore:CHAT_MSG_ADDON(prefix, datastr, scope, sender)
 	if COMM_NAME == prefix then
 		-- Get the command
 		local command, data = string.split(COMM_COMMAND_DELIM, datastr)
+		if DEPRECATED_COMMANDS[command] then return end
+
+		-- Handle weird commands
+		if alert_msg_time[command] == nil then
+			local debug_info = {command, data, sender}
+			table.insert(Hardcore_Settings.debug_log, debug_info)
+			alert_msg_time.UNKNOWN[sender] = time()
+			-- Display that someone is trying to send unknown messages; notifies mods to look at saved_vars and remove player from guild
+			Hardcore:Monitor("|cffFF0000Received unknown messages from " .. sender .. ".")
+			return
+		end
+
+		if alert_msg_time[command][sender] and (time() - alert_msg_time[command][sender] < COMM_SPAM_THRESHOLD[command]) then
+			local debug_info = {command, data, sender}
+			table.insert(Hardcore_Settings.debug_log, debug_info)
+			alert_msg_time[command][sender] = time()
+			-- Display that someone is trying to send spam messages; notifies mods to look at saved_vars and remove player from guild
+			Hardcore:Monitor("|cffFF0000Received spam from " .. sender .. ", using the " .. command .. " command.")
```

Comment:
Maybe we should add thorttling for this monitor message. Otherwise the chat will be spammed when somebody spams addon chat messages.

Reply:
Added throttles!